t/cobrand/southwark.t

```diff
@@ -71,17 +71,8 @@ FixMyStreet::override_config {
         my $json = $mech->get_ok_json(
             '/report/new/ajax?longitude=-0.08051&latitude=51.50351');
 
-        is_deeply $json->{by_category} => {
-            'Abandoned Bike (Street)' => {
-                allow_anonymous => 'true',
-                bodies => ['Southwark Council'],
-            },
-            'Bus Stops and Shelters' => {
-                allow_anonymous => 'true',
-                bodies => ['TfL'],
-            },
-        }, "Southwark 'street' area doesn't have River Piers category";
-
+        is_deeply [sort keys %{$json->{by_category}}], ['Abandoned Bike (Street)', 'Bus Stops and Shelters'],
```

Comment:
Hmm, ideally we wouldn't send the default message with every ajax response, seems a waste of bandwidth if it's not necessary (and as we're storing the original already). I like how it's simplified the templates that use them though. Could we perhaps leave the Default.pm functions as they were, returning nothing, and change report/new/_form_labels.html to, I think, this:
```
SET form_title = c.cobrand.new_report_title_field_label.defined ? c.cobrand.new_report_title_field_label.defined : loc("Summarise the problem");
SET form_title_placeholder = c.cobrand.new_report_title_field_hint.defined ? c.cobrand.new_report_title_field_hint : loc("e.g. ‘10 inch pothole on Example St, near post box’");
SET form_detail_placeholder = c.cobrand.new_report_detail_field_hint.defined ? c.cobrand.new_report_detail_field_hint : loc('e.g. ‘This pothole has been here for two months and…’');
```
That would then have the page defaults in the one place, but the ajax calls would be empty as before.